app/src/androidTest/java/org/simple/clinic/di/TestDataSyncOnApprovalModule.kt

```diff
@@ -1,19 +0,0 @@
-package org.simple.clinic.di
-
-import org.simple.clinic.sync.DataSync
-import org.simple.clinic.sync.DataSyncOnApprovalModule
-import org.simple.clinic.sync.IDataSyncOnApproval
-import org.simple.clinic.user.UserSession
-import org.simple.clinic.util.scheduler.SchedulersProvider
-
-class TestDataSyncOnApprovalModule : DataSyncOnApprovalModule() {
```

Comment:
Curious, why are we removing this?

Reply:
This was being replaced earlier because `TestClinicApp` extended `ClinicApp`, which also sets up triggers like sync everything when users get logged in, etc. This ended up causing those triggers to also run in tests, which was unnecessary and which might also interfere with the test runs themselves. So we added this replacement no-op modules to not do anything in the tests.

In this commit, we changed `TestClinicApp` to extend `Application` instead of `ClinicApp`, which means none of these triggers will ever run, so replacing them with no-ops in unnecessary.